.claude/skills/document-skills/docx/ooxml/scripts/validation/docx.py

```
@@ -0,0 +1,274 @@
+                with zipfile.ZipFile(self.original_file, "r") as zip_ref:
+                    zip_ref.extractall(temp_dir)
+
+                # Parse document.xml
+                doc_xml_path = temp_dir + "/word/document.xml"
+                root = lxml.etree.parse(doc_xml_path).getroot()
+
+                # Count all w:p elements
+                paragraphs = root.findall(f".//{{{self.WORD_2006_NAMESPACE}}}p")
+                count = len(paragraphs)
+
+        except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)